src/index.ts

```diff
@@ -64,6 +64,8 @@ export { default as StepFunctionGraph } from './StepFunctionGraph/Graph';
 
 export { default as StepFunctionRenderer } from './StepFunctionRenderer/StepFunctionRenderer';
 
+export { default as Tabs } from './Tabs';
```

Comment:
You might also export the associate Tab typescript types (specifically the TabDef) I believe. See how other types are exported in this file if you need a reference.